mcan/src/bus.rs

```diff
@@ -157,6 +157,14 @@ pub trait DynAux {
     /// disable CAN operation.
     fn initialization_mode(&self);
 
+    /// Sets MCAN up for being shut down.
+    /// See `Power Down (sleep mode)`, in the peripheral docs, for user
+    /// consideration regarding clocking and message handling.
+    fn shutdown(&self);
```

Comment:
To keep the terminology similar to the users manual and the rest of the API, what about `powerdown_mode`?

Reply:
I agree with both your points. For that reason, `power_down_mode` seems good, since it makes it clearer that it's not an action, but a state. I've thought about `into_power_down_mode`, but I don't think it makes it that much clearer. 

"Sleep mode" is also used in the official documentation interchangeably with "power down mode", so we could stick with that as well if we want a shorter name. But, "power down mode" is more frequently used than "sleep mode".

Reply:
> Hmm, that is a good point. But now that I think about it; it's a bit misleading to label it as `shutdown_mode` or `powerdown_mode` since the device is still configurable and you can queue messages on the tx buffers. In other words, we have not "powered down" the peripheral when we are in `powerdown_mode`.
> 
> Or I might just be overthinking this, what do you think?

You are right that you can still do that, but the device is in a _mode_ that the manual refer to as "Power Down" mode. I think we update function's docstring to mention that this mode does not power down anything, just that refers to the MCAN "Power Down" mode and that it is ready to be actually powered down.